webpack.production.config.js

```diff
@@ -20,17 +20,18 @@ module.exports = {
 
   plugins: [
     new CleanWebpackPlugin(['dist']),
-    new webpack.optimize.OccurenceOrderPlugin(),
     new HtmlWebpackPlugin({
       template: 'src/index.tpl.html',
       inject: 'body',
       filename: 'index.html',
       gtm: '<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WDW6V4" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({"gtm.start":new Date().getTime(),event:"gtm.js"});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!="dataLayer"?"&l="+l:"";j.async=true;j.src="//www.googletagmanager.com/gtm.js?id="+i+dl;f.parentNode.insertBefore(j,f);})(window,document,"script","dataLayer","GTM-WDW6V4");</script>',
     }),
-    new ExtractTextPlugin('[name]-[hash].min.css', { allChunks: true }),
+    new ExtractTextPlugin({
+      filename: '[name]-[hash].min.css',
+      allchunks: true,
+    }),
     new webpack.optimize.UglifyJsPlugin({
       compressor: {
-        warnings: false,
```

Comment:
This defaults to false now. https://webpack.js.org/guides/migrating/#uglifyjsplugin-warnings